src/App.jsx

```diff
@@ -1,40 +1,44 @@
-import React from 'react';
-import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
+import 'bulma/css/bulma.css';
+import { useState } from 'react';
 import './App.scss';
+import { Tabs } from './components/Tabs/Tabs';
 
 export const tabs = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">
-      Selected tab is Tab 1
-    </h1>
-
-    <div data-cy="TabsComponent">
-      <div className="tabs is-boxed">
-        <ul>
-          <li className="is-active" data-cy="Tab">
-            <a href="#tab-1" data-cy="TabLink">Tab 1</a>
-          </li>
-
-          <li data-cy="Tab">
-            <a href="#tab-2" data-cy="TabLink">Tab 2</a>
-          </li>
-
-          <li data-cy="Tab">
-            <a href="#tab-3" data-cy="TabLink">Tab 3</a>
-          </li>
-        </ul>
-      </div>
-
-      <div className="block" data-cy="TabContent">
-        Some text 1
-      </div>
+export const App = () => {
+  const getCurrentTab = (tabID) => {
+    const newTab = tabs.find(tab => tab.id === tabID);
+
+    if (!newTab) {
+      setSelectedTabId(tabs[0].id);
+
+      return getCurrentTab(tabs[0].id);
+    }
+
+    return newTab;
+  };
+
+  const setNewTab = (tab) => {
```

Comment:
if a function only needs the entity ID, it is better to specify only the ID in the parameters
```const setNewTab = (tabId) => {```